Keyguard.d.ts

```diff
@@ -23,3 +23,5 @@ type AccountEntry = {
     type: string,
     label: string
 }
+
+interface Window { rpcServer: any; KeyStore: any }
```

Comment:
Is this really necessary? If it is, then KeyStore should not be `any`, should it? It has type `KeyStore`.

Reply:
doesn't really affect type checking